src/guide/install.md

```diff
@@ -62,4 +62,4 @@ dependencies:
 ```
 ## Python
 
-Currently python is not acceable as a package and is needed to build it from soruce. Please visit [sdk-core](https://github.com/breez/breez-sdk/tree/main/libs/sdk-bindings#python) project for instructions.
+Currently python is not acceable as a package and is needed to build it from source. Please visit [sdk-core](https://github.com/breez/breez-sdk/tree/main/libs/sdk-bindings#python) project for instructions.
```

Comment:
Worth adding the instructions to install using nugets in the c# section.

Reply:
Yeah, the nuget is not available yet though, so wanted to wait until it was published. It needs to be tested on Windows and Linux.